blog-meeting-minutes/2024-03-08-office-hour.md

```diff
@@ -0,0 +1,44 @@
+---
+slug: office-hour-2024-03-08
+title: Office Hour 08.03.2024
+authors: 
+    - almadi_gabor
+tags: [meeting-minutes, community]
+---
+
+## Office Hour meeting minutes
+
+### System team
+
+- n/a
+
+### Security team
+
+- TRG [8.01](https://eclipse-tractusx.github.io/docs/release/trg-0/trg-8-01),
+  [8.03](https://eclipse-tractusx.github.io/docs/release/trg-0/trg-8-03),
+  [8.04](https://eclipse-tractusx.github.io/docs/release/trg-0/trg-8-04),
+  [8.05](https://eclipse-tractusx.github.io/docs/release/trg-0/trg-8-05) 
+  first drafts are created, final versions will come soon
+- Be patient with [CodeQL](https://github.com/eclipse-tractusx/sig-security/blob/main/security-tooling.md#codeql), could be tedious since it does provide a lot of findings
+
+### FOSS
+
+- New [commiter election](https://projects.eclipse.org/projects/automotive.tractusx/elections/election-rohan-krishnamurthy-committer-eclipse-tractus-x) is open for Rohan Krishnamurthy. Please visit the page and make your vote!
+
+### Open planning / community
+
+- Role of the committer is being discussed, it will be presented in the next committer meeting.
+  Basic role descriptions come from the Eclipse Foundation, but we want to specify in Tractus-X
+  what else can be expected from a [contributor](https://www.eclipse.org/legal/ECA.php),
+  [commiter](https://www.eclipse.org/membership/become_a_member/committer.php) and project lead.
```

Comment:
```suggestion
  [commiter](https://www.eclipse.org/membership/become_a_member/committer.php) and [project lead](https://www.eclipse.org/projects/handbook/#roles-pl).
```

maybe some more links which roles and responsibilities we have.

Reply:
Good point, I did not find this document.